TelCo.ColorCoder/NewFeature.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Globalization;
+namespace Telco.ColorCoder
+{
+    class NewFeature
+    {
+        public static string GetManual(int numberOfMajorColors, int numberOfMinorColors)
+        {
+            CultureInfo provider = new CultureInfo("en-us");
+            int outerLoopCount = 0, innerLoopCount = 0, pairNumber = 1;
+            string allPairs = "";
+            for (outerLoopCount = 0; outerLoopCount < numberOfMajorColors; outerLoopCount++)
+            {
+                for (innerLoopCount = 0; innerLoopCount < numberOfMinorColors; innerLoopCount++)
+                {
+                    ColorCode.ColorPair colorPair = ColorMap.GetColorFromPairNumber(pairNumber);
+                    string temp =pairNumber.ToString(provider)+ " - " + colorPair.ToString();

```

Comment:
Any reason to use temp here? Can't we do allPairs += in every line? It will keep appending.

Reply:
Well, temp is not needed as of now but considering the case when one or more particular number - pair maps need to be excluded or have some functionality done, string allPairs shouldn't be bothered.
All it does is, take in a map at a time and append to itself.
If any change in the format of map or anything else, the functionality or work done by/to allPairs would remain unchanged so that if it is required, that one line 23 can be modified into a function call.

Reply:
Good reasoning to isolate the formatting of single lines 👍 
Such isolation can be done in a function with an appropriate name, as you have indicated.

Reply:
Yes. I suppose I could do that. Thanks.